package.json

```diff
@@ -159,7 +159,7 @@
     "htmlhint": "^0.11.0",
     "jsdom": "^15.1.1",
     "localstorage-memory": "^1.0.3",
-    "mediasoup-client": "github:versatica/mediasoup-client#v3",
+    "mediasoup-client": "^3.6.21",
```

Comment:
Nice call updating this, we should definitely try to stay on top of bugfixes upstream and keep it up to date (though its hard to do so with lots of confidence we arent breaking things). This debug panel should help, as should QA process.

Reply:
Yep, the version we were using was pretty old and this seems like a good moment to bump it.